scripts/test/run-tests.sh

```diff
@@ -60,20 +55,32 @@ if [ "$1" == "--prebuilt-image" ]; then
 
 else
   if [ "$1" == "--all" -o "$1" == "--only" -a "$2" == "qemux86_64" ]; then
+    wget --progress=dot:giga -N ${UBUNTU_IMAGE_URL} -P input/
     convert_and_test "qemux86_64" \
                      "release-1" \
-                     "${UBUNTU_IMAGE_URL}" \
-                     "${UBUNTU_IMAGE}" \
-                     "${UBUNTU_IMAGE}.gz" \
-                     "configs/qemux86-64_config" || test_result=$?
+                     "input/Ubuntu-Bionic-x86-64.img.gz" \
```

Comment:
Why hardcoded, would not `$(basename ${UBUNTU_IMAGE_URL})` work?

Reply:
Because this way it mirrors the uncompressed input image, which does not match the basename of the input. This is easy to fix though, if you prefer something like:
`$(basename ${UBUNTU_IMAGE_URL##.*})` for the uncompressed test.

Reply:
No strong opinions